R/tv_remote.R

```diff
@@ -1,23 +1,25 @@
 #' @export
-tv_remote <- function() {
+tv_remote <- function(tv_path, tv_port) {
```

Comment:
Do we need to document arguments? Perhaps use `path` and `port` only?

Reply:
we won't export `tv_remote`, only `on()`, `off()`, and perhaps `tv_status()` -> `status()`, and `tv_path()` -> `path()`, at least for now, because they may be useful for debugging. 

Will clean up in a later PR.

Reply:
There is rarely a reason for writing `library(tv)`, so these functions will always be called as `tv::on()`, `tv::status()`, etc. Thus, we don't need the `tv_` prefix. Same goes for the arguments - no prefix.